src/energy_comms/transform/cps.py

```diff
@@ -0,0 +1,50 @@
+# imports
+
+import json
+
+import pandas as pd
+import prettytable
+import requests
+
+
+def transform(x):
+    # convert API pretty table to pandas dataframe
+
+    tbl_as_csv = x.get_csv_string().replace("\r", "")
+
+    txt = open("national_unemployment.csv", "w")
+
+    n = txt.write(tbl_as_csv)
+    txt.close()
+
+    cps = pd.read_csv("national_unemployment.csv")
```

Comment:
If we go straight to Pandas dataframe in the extract, then this part where we go to CSV isn't necessary. We can pass in `raw_df` to this function and start here. 

The first thing we want to do is enforce data types so we can safely do things like `groupby`. 
```
df = df.dropna(subset=["year"])
df = df.astype({"year": "int", "period": "string", "periodName": "string", "value": "float", "series_id":"string"})
```